custom/icds_reports/mpr_sqldata.py

```diff
@@ -412,6 +414,157 @@ def row_config(self):
         )
 
 
+
+class MPRSupplementaryNutritionBeta(ICDSMixin, MPRData):
+
+    title = '4. Delivery of Supplementary Nutrition and Pre-School Education (PSE)'
+    slug = 'supplementary_nutrition'
+
+    def __init__(self, config, allow_conditional_agg=False):
+        super(MPRSupplementaryNutritionBeta, self).__init__(config, allow_conditional_agg)
+        self.awc_open_count = 0
+
+    @property
+    def subtitle(self):
+        return 'Average no. days AWCs were open during the month? %.1f' % (
+            self.awc_open_count / (self.awc_number or 1)
+        ),
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(''),
+            DataTablesColumn(_('Morning snacks/ breakfast')),
+            DataTablesColumn(_('Hot cooked meals/RTE')),
+            DataTablesColumn(_('Take home ration (THR')),
+            DataTablesColumn(_('PSE'))
+        )
+
+    @property
+    def rows(self):
+        if self.config['location_id']:
+            filters = get_location_filter(self.config['location_id'], self.config['domain'])
+            if filters.get('aggregation_level') > 1:
+                filters['aggregation_level'] -= 1
+
+            filters['month'] = date(self.config['year'], self.config['month'], 1)
+
+            data = ServiceDeliveryReportView.objects.filter(**filters).values(
```

Comment:
It looks like all of these columns are new, how did you choose this report over another one?

Reply:
Assuming by the report you mean the table: I first chose agg_Awc. But it has already very wide. The next option I saw was SDD because it was already capturing info about service deliveries given to kids. That is why I chose SDD.